Source/Urho3D/Graphics/Batch.cpp

```diff
@@ -781,24 +781,25 @@ void BatchQueue::SortFrontToBack()
     for (auto i = batchGroups_.begin(); i != batchGroups_.end(); ++i)
         sortedBatchGroups_[index++] = &i->second;
 
-    SortFrontToBack2Pass(reinterpret_cast<ea::vector<Batch*>& >(sortedBatchGroups_));
+
+    SortFrontToBack2Pass(reinterpret_cast<Batch**>(sortedBatchGroups_.begin()), reinterpret_cast<Batch**>(sortedBatchGroups_.end()) );
```

Comment:
This should be `static_cast<>`. Not that `reinterpret_cast<>` is wrong or wont work. Using right cast is more kosher :]

Reply:
I tried changing it to static_cast, I get the error:

'static_cast': cannot convert from 'Urho3D::BatchGroup **' to 'Urho3D::Batch **'	Urho3D	C:\Users\casht\Repos\techgame\rbfx\Source\Urho3D\Graphics\Batch.cpp	785

Reply:
Looks like it is due to it casting a double pointer and not a single pointer.  
https://stackoverflow.com/questions/26277391/why-cant-static-cast-a-double-void-pointer